src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -11,47 +11,55 @@ public static class LocalDBTest
         private static bool IsLocalDBEnvironmentSet() => DataTestUtility.IsLocalDBInstalled();
 
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // No Registry support on UAP
-        [ConditionalFact(nameof(IsLocalDBEnvironmentSet))]
-        public static void LocalDBConnectionTest()
+        [ConditionalTheory(nameof(IsLocalDBEnvironmentSet))]
+        [MemberData(nameof(LocalDbSourceProvider))]
+        public static void LocalDBConnectionTest(string connectionString)
         {
-            SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(@$"server=(localdb)\{DataTestUtility.LocalDbAppName}");
-            builder.IntegratedSecurity = true;
-            builder.ConnectTimeout = 2;
+            SqlConnectionStringBuilder builder = new(connectionString)
+            {
+                IntegratedSecurity = true,
```

Comment:
This test should only be run if "SupportsIntegratedSecurity" is true.

Reply:
done.